packages/create-astro/src/messages.ts

```diff
@@ -0,0 +1,101 @@
+/* eslint no-console: 'off' */
+import { exec } from 'node:child_process';
+import { get } from 'node:https';
+import { color, label } from '@astrojs/cli-kit';
+import { sleep } from '@astrojs/cli-kit/utils';
+import stripAnsi from 'strip-ansi';
+
+export const welcome = [
+		`Let's claim your corner of the internet.`,
+    `I'll be your assistant today.`,
+    `Let's build something awesome!`,
+    `Let's build something great!`,
+    `Let's build something fast!`,
+    `Let's make the web weird!`,
+    `Let's make the web a better place!`,
+    `Let's create a new project!`,
+    `Let's create something unqiue!`,
+    `Time to build a new website.`,
+    `Time to build a faster website.`,
+    `Time to build a sweet new website.`,
+    `We're glad to have you on board.`,
+    `Keeping the internet weird since 2021.`,
+    `Initiating launch sequence...`,
+    `Initiating launch sequence... right... now!`,
+    `Awaiting further instructions.`,
+]
+
+export const getName = () => new Promise((resolve) => {
+  exec('git config user.name', { encoding: 'utf-8' }, (_1, gitName, _2) => {
+    if (gitName.trim()) {
+        return resolve(gitName.split(' ')[0].trim());
+    }
+    exec('whoami', { encoding: 'utf-8' }, (_3, whoami, _4) => {
+        if (whoami.trim()) {
+            return resolve(whoami.split(' ')[0].trim());
+        }
+        return resolve('astronaut');
+    });
+  });
+});
+
+let v: string;
+export const getVersion = () => new Promise<string>((resolve) => {
+    if (v) return resolve(v);
+    get('https://registry.npmjs.org/astro/latest', (res) => {
+        let body = '';
+        res.on('data', chunk => body += chunk)
+        res.on('end', () => {
+            const { version } = JSON.parse(body);
+            v = version;
+            resolve(version);
+        })
+    })
+})
+
+export const banner = async (version: string) => console.log(`\n${label('astro', color.bgGreen, color.black)}  ${color.green(color.bold(`v${version}`))} ${color.bold('Launch sequence initiated.')}\n`);
```

Comment:
nit: I believe we still try to do `function fn` over `const fn =` when we can. I tried searching for `function info` in this PR to find this and it took me a bit longer to find because of it.